frontend/src/pages/Overview/actions.js

```diff
@@ -75,12 +78,26 @@ export function showProjectPermissions(id) {
   };
 }
 
+export function showProjectAdditionalData(id) {
+  return {
+    type: SHOW_PROJECT_ADDITIONAL_DATA,
+    id
+  };
+}
+
 export function hideProjectPermissions() {
   return {
     type: HIDE_PROJECT_PERMISSIONS
   };
 }
 
+export function hideProjectAdditionalData(id) {
+  return {
+    type: HIDE_PROJECT_ADDITIONAL_DATA,
+    id
```

Comment:
id is unused, i guess it is not necessary.

Reply:
correct, removed